[Cache] Adding Cache content from KB (#1636)
cloudflare/cloudflare-docs: products/cache/src/content/best-practices/always-online.md

---
title: Always Online
order: 16
pcx-content-type: concept
---

# Always Online

Observe the following best practices when enabling Always Online with Internet Archive integration.

> - **Allow requests from Cloudflare IP addresses.** Origin servers for domains [proxied through Cloudflare](https://support.cloudflare.com/hc/articles/115005138088) receive requests from Cloudflare IP addresses. To avoid blocking these requests, follow the guidelines for [allowing Cloudflare IP addresses](https://support.cloudflare.com/hc/articles/201897700). When you observe the Always Online banner while your origin web server is online, your origin web server or hosting provider is likely blocking or rate limiting Cloudflare requests.

Reviewer: ```suggestion
- **Allow requests from Cloudflare IP addresses.** Origin servers for domains [proxied through Cloudflare](https://support.cloudflare.com/hc/articles/205195708#h_815d1b1e-74cf-4a5f-b477-e441a76ac3fd) receive requests from Cloudflare IP addresses. To avoid blocking these requests, follow the guidelines for [allowing Cloudflare IP addresses](https://support.cloudflare.com/hc/articles/201897700). When you observe the Always Online banner while your origin web server is online, your origin web server or hosting provider is likely blocking or rate limiting Cloudflare requests.
```

Reviewer: Believe this is the right place. The old one sent me to LB docs.

- **Configure your origin server’s Cache Control header.** To ensure Always Online caches resources for your site, do not set your origin server’s Cache Control header to `no-cache`, `must-revalidate`, or `max-age=0`.
- **Consider potential conflicts with Cloudflare features that transform URIs.** Always Online Beta with Internet Archive integration may cause issues with Page Rules and other Cloudflare features that transform URIs due to the way the Internet Archive crawls pages to archive. Specifically, some redirects that take place at the edge may cause the Internet Archive's crawler not to archive the target URL. Before enabling Origin Cache Control, review [how Cloudflare caches resources by default](/about/default-cache-behavior) as well as any Page Rules you have configured so that you can avoid these issues. If you experience problems, disable Always Online Beta.

Do not use Always Online with:

> - [Custom Hostnames (SSL for SaaS)](https://support.cloudflare.com/hc/articles/217371987)

Reviewer: ```suggestion
- [Custom Hostnames (SSL for SaaS)](https://developers.cloudflare.com/ssl/ssl-for-saas)
```

- API traffic
- An [IP Access Rule](https://support.cloudflare.com/hc/articles/217074967) or [Firewall Rule](https://support.cloudflare.com/hc/articles/360016473712) that blocks the United States or
- A [Cache Everything Page Rule](/how-to/create-page-rules#cache-everything) that configures an Edge Cache TTL lower than the Always Online crawl frequency pertaining to your domain plan type.

## Limitations

There are limitations with the Always Online functionality: 

1. Always Online is not immediately active for sites recently added to Cloudflare due to:

>     - DNS record propagation (can take 24-72 hours), or

Reviewer: ```suggestion
    - DNS record propagation (can take 24-72 hours)
```

>     - Always Online has not initially crawled the website. 

Reviewer: ```suggestion
    - Always Online has not initially crawled the website. 
```
```suggestion
    - Always Online has not initially crawled the website
```
